indico/modules/events/editing/schemas.py

```diff
@@ -139,6 +140,29 @@ class Meta:
     review_conditions_valid = fields.Boolean()
 
 
+class EditableBasicSchema(mm.ModelSchema):
+    class Meta:
+        model = Editable
+        fields = ('id', 'type', 'state', 'editor')
+
+    state = fields.String()
+    editor = fields.Nested(UserSchema, only=('id', 'avatar_bg_color', 'full_name'))
+
+
+class EditingEditableListSchema(mm.ModelSchema):
+    class Meta:
+        model = Contribution
+        rh_context = ('editable_type')
+        fields = ('id', 'friendly_id', 'title', 'code', 'editable')
+
+    editable = fields.Method('_get_editable')
+
+    def _get_editable(self, contribution):
+        editable_type = self.context['editable_type']
+        editable = {editable for editable in contribution.editables if editable.type.name == editable_type.name}
```

Comment:
why are you comparing by name here?

Reply:
Comparing whether it is a `paper`, `slides` or `poster`.

Reply:
What I mean is, why not just compare `editable.type == editable_type`